src/sage/numerical/backends/cvxpy_backend.pyx

```diff
@@ -932,3 +932,5 @@ cdef class CVXPYBackend:
             self.col_lower_bound[index] = value
         else:
             return self.col_lower_bound[index]
+    def remove_constraint(self, i):
+        del self.constraints[i]
```

Comment:
You will want to look at the method `add_linear_constraint`. What that method adds, this method needs to remove.